runtime/src/bank.rs

```diff
@@ -3583,6 +3610,31 @@ impl Bank {
         self.log_epoch_rewards_sysvar("update");
     }
 
+    /// Create the persistent PDA containing the epoch-rewards data
+    fn create_epoch_rewards_partition_list_account(
```

Comment:
is this going to be a different account from the one that will hold the rest of the distribution period parameters, or am i reviewing ahead of the work that's been done?

Reply:
What other parameters are you referring to? I was thinking with known number of partitions, we were set, but are you wanting the fields in [PartitionedEpochRewardsConfig](https://github.com/solana-labs/solana/blob/09a94b81977b1d909e20f110f94ed11684ddf353/accounts-db/src/partitioned_rewards.rs#L8)? Or something else? Could certainly add them.

Reply:
i guess we implicitly have the number of partitions from the length of the vec... was thinking the parent blockhash as per previous comment, but may be able to forego if i can't recover the rationale

Reply:
Yeah, if we punt on the distribution hash, I think the account could just contain num_partitions and parent blockhash.